<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiAddPluginDialog</class>
 <widget class="QDialog" name="UiAddPluginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new Plug-in</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QDialog {
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="webGroupbox">
     <property name="styleSheet">
      <string notr="true">QGroupBox::title {
	image: url(:/AddPluginDialog/Web);
	image-position : right;
}</string>
     </property>
     <property name="title">
      <string>Web plugin     </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Plugin url :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="webUrl"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localGroupbox">
     <property name="styleSheet">
      <string notr="true">QGroupBox::title
{
	image: url(:/AddPluginDialog/Folder);
	image-position : right;
}</string>
     </property>
     <property name="title">
      <string>Local plugin     </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Plugin path :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="localUrl"/>
        </item>
        <item>
         <widget class="QToolButton" name="localBrowse">
          <property name="styleSheet">
           <string notr="true"> QToolButton {
     border: none;
     background-color: none;
 }</string>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="icon">
           <iconset resource="Icons.qrc">
            <normaloff>:/AddPluginDialog/Browse</normaloff>:/AddPluginDialog/Browse</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="ok">
       <property name="styleSheet">
        <string notr="true"> QToolButton {
     border: none;
     background-color: none;
 }</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/AddPluginDialog/Ok</normaloff>:/AddPluginDialog/Ok</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancel">
       <property name="styleSheet">
        <string notr="true"> QToolButton {
     border: none;
     background-color: none;
 }</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/AddPluginDialog/Cancel</normaloff>:/AddPluginDialog/Cancel</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>webUrl</tabstop>
  <tabstop>localGroupbox</tabstop>
  <tabstop>localUrl</tabstop>
  <tabstop>localBrowse</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>webGroupbox</tabstop>
 </tabstops>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>UiAddPluginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>UiAddPluginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
